frontend/.gitignore

```diff
@@ -12,6 +12,9 @@ Notes.txt
 # production
 /build
 
+#alex's stuff
```

Comment:
Probably best if you add this kind of thing to your Global git ignore and keep it out of the project. 

https://help.github.com/articles/ignoring-files/